strapi/strapi
# [Blocks editor] Remove empty line before block without editor selection

### What does it do?

It removes the empty line before a block content created when there is not editor.selection

### Why is it needed?

Because we don't wont an empty line

### How to test it?

Create a new instance of a collection with the block field and inside the block field select directly the code option (or every other block option), then you need to see the new code block without an empty line before it
